Evaluator/RealTime/instant_fluctuations_evaluator.py

```diff
@@ -2,7 +2,6 @@
 OctoBot Tentacle
 
 $tentacle_description: {
-    "package_name": "OctoBot-Tentacles",
```

Comment:
Why did you remove this line ?

Reply:
Because if was a mistake to add it in the first place, it's an auto generated field by the tentacle manager